packages/components/src/Modal/GenericModal/GenericModal.module.scss

```diff
@@ -54,6 +54,20 @@
   @include ca-media-tablet-and-up {
     width: 100%;
   }
+
+  // wrap the modal container with a focus ring when the title has focus
+  // don't copy this over on the rebuild
+  // rather than putting the focus on the title, put it on the role=dialog element itself
+  &:has(:global([class*="modalLabel"]):focus-visible)::after {
```

Comment:
🧙‍♂️

Reply:
I don't see a ring when I go to ContextModal stories 🙈

Reply:
Maybe it's being clipped somehow
![image](https://github.com/user-attachments/assets/e1b51891-1805-4e02-b5fb-f2a069e5ac9c)

But Chromatic shows it!

Reply:
It's `focus-visible` so you'll need to not use the mouse 😉

Reply:
Ohhh duh

Reply:
fyi I tried a couple of things to avoid getting this hacky:
- tried moving the focus to the dialog instead of the title
  - unfortunately the styling is put on an inner child of the dialog, so that would require more work
- tried giving this element an extra class from the GenericModal.module.scss file so that I could target that instead of using `:global` with a wildcard, but it didn't work for some reason.

but it's gonna be legacy code so i'm saying good enough